docs/guides/security/authentication/secure-instance-with-fido2-key/index.md

```diff
@@ -0,0 +1,228 @@
+
+[Akamai MFA](https://www.akamai.com/products/akamai-mfa) is a full-service MFA solution for the workforce, featuring a phishing-proof FIDO2 authentication factor. However, the full Akamai MFA service is not required in order to support MFA for your Compute Instances. Rather, you only need the free Akamai MFA mobile app and Akamai's Krypton akr SSH agent. Krypton is an SSH agent that cooperates with the Akamai MFA mobile app to perform FIDO2-based authentication. Akamai akr serves as a bridge between your local machine and the Akamai MFA mobile app on your phone, while the Akamai MFA mobile app turns your phone into a FIDO2 authenticator. The FIDO2 private key that serves as your credential is securely stored on your phone (and never leaves the phone) while the actual challenge is presented to you as a phone notification.
+
+## Before You Begin
+
+-   **Install the Akamai MFA mobile app** on your phone or tablet using the [Google Play Store](https://play.google.com/store/apps/details?id=com.akamai.pushzero&hl=en_US&gl=US) (for Android) or the [App Store](https://apps.apple.com/us/app/akamai-mfa/id1503619894) (for iOS).
+
+-   **Verify you are running a supported operating system**, which includes macOS 10.15+, Debian 11+, Ubuntu 20.04+, AlmaLinux 9, and Rocky Linux 9. Older versions of some Linux distributions may work *if* the OpenSSH package is upgraded to match the version requirement listed below. Upgrading OpenSSH is outside the scope of this guide.
+
+-   **Both your local and remote systems must use OpenSSH 8.2+.** The Akamai akr tool generates keys using the ECDSA algorithm with 256-bit strength and outputs them in the `sk-ecdsa-sha2-nistp256` format. Due to this, both the local system and any remote systems that use the public key must have OpenSSH 8.2 (or greater) installed. To check your OpenSSH version, run:
+
+    ```command
+    ssh -V
+    ```
+
+## Installation Instructions
+
+Instructions on installing the akr tool (which includes Akamai's Krypton SSH Agent) to your system. For additional installation help, review the [official instructions](https://techdocs.akamai.com/mfa/docs/akr-fido2-ssh-agent#install-or-update-your-akr-fido2-ssh-agent) on Akamai's documentation site.
+
+{{< tabs >}}
+{{< tab "macOS 10.15+" >}}
+To start using FIDO2 authentication with Akamai akr, macOS users can install the agent with [Homebrew](https://brew.sh/).
+
+1.  The OpenSSH version include on macOS Ventura (and earlier releases) does not support the necessary security keys features. To overcome this, install OpenSSH through Homebrew and update your PATH variable so that this new version is used instead of the default macOS version.
```

Comment:
```suggestion
1.  The OpenSSH version included on macOS Ventura (and earlier releases) does not support the necessary security keys features. To overcome this, install OpenSSH through Homebrew and update your PATH variable so that this new version is used instead of the default macOS version.
```